Makefile

```diff
@@ -127,6 +127,9 @@ $(DOCKER_DIRS):
 systemtests:
 	make -C systemtests
 
+tekton-test:
```

Comment:
Maybe name it systemtests-tekton to make the names a bit more similar and logical?

Reply:
Renamed